README.md

```diff
@@ -23,7 +23,7 @@ pip install unfollow
 
 ### Usage
 
-The first time you run the tool, you will be asked to enter your GitHub username. Subsequent runs will show any unfollowers, with names and links.  
+You will need to enter your username when you will the program for the first time. Subsequent runs will show any unfollowers, with names and links.  
```

Comment:
You forgot `run` after `you will`.
Furthermore, I think that this version is clearer.